src/Apps/Comparables/Filter/AttributionClassFilter.tsx

```diff
@@ -1,42 +1,59 @@
+import { Checkbox } from "@artsy/palette"
 import { FilterState } from "Apps/Collect/FilterState"
+import { ContextConsumer } from "Artsy/SystemContext"
 import React from "react"
+import { Subscribe } from "unstated"
 
-import { Checkbox } from "@artsy/palette"
-import { SystemProps } from "Artsy/SystemContext"
-
-export const AttributionClassFilter: React.SFC<{
-  filters: FilterState
-  mediator: SystemProps["mediator"]
-}> = ({ filters, mediator }) => (
-  <>
-    {Object.keys(AttributionClasses).map((attributionClass, index) => {
-      const className = AttributionClasses[attributionClass].name
-      const isSelected = filters.state.attribution_class.includes(
-        attributionClass
-      )
+export const AttributionClassFilter: React.SFC = () => {
+  return (
+    <ContextConsumer>
+      {({ mediator }) => {
+        return (
+          <Subscribe to={[FilterState]}>
+            {(filters: FilterState) => {
+              return (
```

Comment:
Nice, now this component can be moved around, refactored, etc, without being locked into a hierarchical contract.